smItems/src/main/kotlin/me/ixhbinphoenix/smPl/smItems/commands/giveitem.kt

```diff
@@ -49,6 +41,7 @@ class giveItemCommand : BaseCommand {
           val item = itemUtils.createArmor(Material.LEATHER_HELMET, "Programmer's Cat ears", "programmers_cat_ears", Rarity.MYTHIC, ArmorTypes.HELMET, 1000, 1000, set)
           val im = item.itemMeta as LeatherArmorMeta
           im.setColor(Color.FUCHSIA)
+          im.isUnbreakable = true
```

Comment:
This is actually a problem, but shouldn't be fixed this way. Instead, change the im.isUnbrekable in itemUtils.createArmor() to true, since it's currently false for some reason. Armor always needs the unbreakable flag, so adding it to createArmor() is a more DRY way of approaching this problem

Reply:
this has been fixed on the latest master commit, and thus, this fix shouldn't be merged into master